asn3/jEdit/org/gjt/sp/jedit/gui/SplashScreen.java

```diff
@@ -160,6 +177,66 @@ public synchronized void paintComponent(Graphics g)
 			getWidth() - fm.stringWidth(version) - 2,
 			image.getHeight(this) - fm.getDescent());
 		notify();
+
+		boolean isAboveProgressBar;
+		int y = scrollPosition;
+		for(String member: groupMembers)
+		{
+		    isAboveProgressBar = y < (INITIAL_SCROLL_HEIGHT - PROGRESS_HEIGHT + 10);
+		    if (isAboveProgressBar)
+				g.drawString(member, (maxWidth - fm.stringWidth(member)) / 2, y);

```

Comment:
It was clashing with the progress-bar and looked gross. This is a hack so it prints only those who are fully above the bar.